### PR TITLE
service health communications API: api and resource, toc files update for attachment

### PR DESCRIPTION
**Pull request by payiAzure**

### CODE REVIEW

**Comment by isvargasmsft** [comment]
Item [12451]

File: api-reference/beta/api/serviceannouncementattachment-get.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+GET /admin/serviceAnnouncement/messages/{serviceUpdateMessageId}/attachments/{serviceAnnouncementAttachmentId}
+```
+
+## Optional query parameters
+This method supports some of the OData query parameters to help customize the response. For general information, see [OData query parameters](/graph/query-parameters).
```
Comment (Lauragra) [inline api-reference/beta/api/serviceannouncementattachment-get.md:37]: Should you revise this boilerplate statement to match that for the list operations - i.e. it supports the OData query parameters (not only some of them).

File: api-reference/beta/api/serviceannouncementattachment-get.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+## Request body
+Do not supply a request body for this method.
+
+## Response
+
+If successful, this method returns a `200 OK` response code and a [serviceAnnouncementAttachment](../resources/serviceannouncementattachment.md) object in the response body.
+
+## Examples
+
+### Example 1: Get an attachment with ID for a message. Return the [serviceAnnouncementAttachment](../resources/serviceannouncementattachment.md) resource.
```
Comment (Lauragra) [inline api-reference/beta/api/serviceannouncementattachment-get.md:53]: We generally don't use multiple complete sentences for headings; headings should be concise, scannable phrases. Suggest this for the heading:

### Example 1: Get an attachment with message ID

Then you can have a description for the example:
The following request returns a **serviceAnnouncementAttachment** resource.

File: api-reference/beta/api/serviceannouncementattachment-get.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+    "isInline": false,
+    "lastModifiedDateTime": "2021-09-21T04:07:15.9720778Z",
+    "name": "An attachment for a Data Privacy message regarding your organization is available within Message Center. The contents of this attachment can be accessed within Message Center by a Global Administrator or someone designated as a Message Center Privacy Reader. Please sign in to Admin Center to view the details of this message in the M365 Message center.",
+    "size": 44583,
+    "id": "30356a46-ffad-47e1-acf6-40a99b1538c1"
+  }
+}
+```
+
+### Example 2: Return a file stream of an attachment content for a message.
```
Comment (Lauragra) [inline api-reference/beta/api/serviceannouncementattachment-get.md:90]: Remove the period from the heading.

File: api-reference/beta/api/serviceannouncementattachment-get.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+  "truncated": true,
+  "@odata.type": "microsoft.graph.serviceAnnouncementAttachment"
+}
+-->
+``` http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "value": "An attachment for a Data Privacy message regarding your organization is available within Message Center. The contents of this attachment can be accessed within Message Center by a Global Administrator or someone designated as a Message Center Privacy Reader. Please sign in to Admin Center to view the details of this message in the M365 Message center."
```
Comment (Lauragra) [inline api-reference/beta/api/serviceannouncementattachment-get.md:115]: Please avoid the abbreviation M365 - spell out as Microsoft 365.

File: api-reference/beta/api/serviceannouncementattachment-get.md (hunk 1 above)
Comment (payiAzure) [reply api-reference/beta/api/serviceannouncementattachment-get.md:37]: will update it

File: api-reference/beta/api/serviceannouncementattachment-get.md (hunk 3 above)
Comment (payiAzure) [reply api-reference/beta/api/serviceannouncementattachment-get.md:90]: will update

File: api-reference/beta/api/serviceannouncementattachment-get.md (hunk 2 above)
Comment (payiAzure) [reply api-reference/beta/api/serviceannouncementattachment-get.md:53]: Done.

File: api-reference/beta/api/serviceannouncementattachment-get.md (hunk 4 above)
Comment (payiAzure) [reply api-reference/beta/api/serviceannouncementattachment-get.md:115]: will update it. But I think it does not matter, this is the attachment content that was posted by the service team, they will put the content as what they would like the customers to see.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**